examples/solid mechanics/viscoplastic-damage/beam/beam.json

```diff
@@ -37,7 +37,7 @@
     "Mesh" : [{
             "Name" : "beam",
             "ConstitutiveModel" : {
-                "Name" : "ChabocheDamage",
+                "Name" : "J2PlasticityDamage",
```

Comment:
What makes more sense here is to keep the overarching yield theory (J2) and append a damage model to that.  So more like
```json
"Name" : "J2Plasticity",
"Damage" : "Isotropic",
or
"Damage" : "Chaboche"
```
That was additional models can be added with ease.